moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -371,12 +399,25 @@ const ProjectTeamTable = ({
                 return personnel[item];
               })
         }
-        title="Project team"
+        title={
+          <Typography variant="h2" color="primary">
+            Project team
+          </Typography>
+        }
         options={{
+          ...(data.moped_proj_personnel.length < 26 && { paging: false }),
           search: false,
           rowStyle: { fontFamily: typography.fontFamily },
           actionsColumnIndex: -1,
         }}
+        localization={{
+          header: {
+            actions: "",
+          },
+          body: {
+            emptyDataSourceMessage: <Typography variant="body1">No team members to display</Typography>,
```

Comment:
@tillyw would you add one of these for the Phases, Milestones, and Files tables? "No phases to display", "No milestones to display", "No files to display".

Reply:
errr, well—I guess we still need to apply all of these customizations to the other material table instances. separate issue @tillyw @amenity ?

Reply:
@johnclary @tillyw yes - sending [this into Slack](https://austininnovation.slack.com/archives/CNUEPKLB1/p1624434006172500).